src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,171 @@
+package duke;
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A list of Task object and methods that deal with modification to the list of tasks.
+ */
+
+public class TaskList {
+
+    private static ArrayList<Task> tasks;
+
+    TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * updating the taskList from the database if file is found
+     * @param listOfTask to be created.
+     */
+    TaskList(List<String> listOfTask) {
+        this.tasks = new ArrayList<>();
+        for (int i = 0; i < listOfTask.size(); i++) {
+            addTaskFromFile(listOfTask.get(i), tasks);
+        }
+    }
+
+    /**
+     * method to mark a task in the list as done
+     * @param remain index of the task in list
```

Comment:
Similar to before, to quote the coding standard, "In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)", so something like "Adds tasks from given string format." will be better, also note the capitalisation and full stop at the end.